roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -145,6 +145,19 @@ reserved_host_memory_mb=0
 {% elif inventory_hostname in groups['controller'] -%}
 reserved_host_memory_mb=4096
 {% endif %}
+
+{% if ceph.enabled -%}
+images_type = rbd
+images_rbd_pool = {{ ceph.vm_pool_name }}
+images_rbd_ceph_conf = {{ ceph.cinder.rbd_ceph_conf }}
+rbd_user = {{ ceph.cinder.rbd_user }}
+rbd_secret_uuid = {{ cinder_uuid_file.content | b64decode }}
+inject_password = false
+inject_key = false
```

Comment:
This(inject_password and inject_key) configurations are not mandatory. Mentioned as a good practice in ceph documentation.
Reference: 
 http://docs.ceph.com/docs/master/rbd/rbd-openstack/#id2